knack/iframeMapMessenger.js

```diff
@@ -76,12 +77,23 @@
       };
 
       sendMessageToApp(markerMessage, locationViewIFrame);
+    }
+    // Location Details Page - Editable
+    $("#view_2609 #mapIFrame").on("load", function() {
+      locationDetailsMapMessage("view_2609");
     });
 
-    $("#mapIFrame").on("load", function() {
+    // Location Details Page - Viewer
+    $("#view_2733 #mapIFrame").on("load", function() {
+      locationDetailsMapMessage("view_2733");
+    });
+
+    // Work Orders Details Page Maps
+    function workOrdersDetialsMapMessage(viewId) {
```

Comment:
Small typo here in the function name (detials instead of details) here and in the calls on line 111 and 115. Doesn't affect the function.